scripts/golangci.yml.in

```diff
@@ -168,3 +168,10 @@ linters:
     - testpackage
     # TODO: enable goerr113, see: https://github.com/ceph/ceph-csi/issues/1227
     - goerr113
+
+issues:
+  exclude-rules:
+    - path: internal/util/conn_pool_test.go
```

Comment:
This is not really nice, maybe some (future) tests in the file can run in parallel? Is it not possible to add a `// noline:...` comment for the function(s) that have the issue?

Reply:
Yeah We can do that.